docs/requirements.txt

```
@@ -5,3 +5,6 @@ myst-parser
 myst-nb
 sphinx-copybutton
 sphinx-autodoc-typehints
+fsspec
+rechunker
```

```suggestion
git+https://github.com/rabernat/rechunker.git@refactor-executors
```

Why don't we go with the dev branch of rechunker for now to match CI, and once that is merged and published we can switch to the latest stable version.

---

```suggestion
git+https://github.com/pangeo-data/rechunker.git
```
The PR has been merged to master